packages/test/test-end-to-end-tests/src/test/summarizeIncrementallySubDds.spec.ts

```diff
@@ -500,6 +503,31 @@ describeCompat(
 			return createSummarizerResult.summarizer;
 		}
 
+		/**
+		 * Validates that the passed summaryObject is a summary handle. Also, the handle id is correct.
+		 */
+		function validateHandle(
+			summaryObject: SummaryObject,
+			dataStoreId: string,
+			ddsId: string,
+			subDDSId: string,
+			messagePrefix: string,
+		) {
+			// The handle id for sub-DDS should be under ".channels/<dataStoreId>/.channels/<ddsId>" as that is where
+			// the summary tree for a sub-DDS is.
+			const expectedHandleId = `/${channelsTreeName}/${dataStoreId}/${channelsTreeName}/${ddsId}/${subDDSId}`;
+			assert.strictEqual(
+				summaryObject.type,
+				SummaryType.Handle,
+				`${messagePrefix} should be a handle`,
+			);
+			assert.strictEqual(
+				summaryObject.handle,
+				expectedHandleId,
+				`${messagePrefix}'s handle id is incorrect`,
```

Comment:
>is incorrect [](http://example.com/codeflow?start=34&length=12)

nit: "should match" just like you have "should be a handle" in 522. The string tells what's expected, the assertion result will say whether expectation was met (Pass) or not(Fail). 

This is what I was trying to convey in my comment in the previous iteration. But this is a nit. Feel free to keep what you have.

Reply:
nvm, I think that paradigm is not followed generally in our tests. Ref 560, 565, 570